abcfold/boltz/run_boltz.py

```diff
@@ -55,7 +55,7 @@ def run_boltz(
         out_file = working_dir.joinpath(f"{input_json.stem}.yaml")
 
         boltz_yaml.write_yaml(out_file)
-        logger.info("Running Boltz1")
+        logger.info("Running Boltz")
```

Comment:
It might be worth changing this to:

`logger.info(f"Running Boltz version: {BOLTZ_VERSION}")`

We could also do something similar for Chai  (As an aside, are we on the latest Chai version? it's 0.6.1?)